framework/src/org/checkerframework/common/value/ReflectiveEvaluator.java

```diff
@@ -94,6 +119,43 @@ public ReflectiveEvalutator(
     }
 
     /**
+     * This method changes a set of varargs invocations (each of which passes an arbitrary number of
```

Comment:
I find the use of "invocation" in this comment confusing.  The "invocation" you are referring to is really not an invocation but an array of arguments.   Using invocation in the name of the method makes it extremely confusing at the call site where the method is actually invoked a few lines after.